core/src/main/java/hudson/model/FingerprintCleanupThread.java

```diff
@@ -50,6 +50,7 @@
 public class FingerprintCleanupThread extends AsyncPeriodicWork {
 
     static final String FINGERPRINTS_DIR_NAME = "fingerprints";
+    public static final Pattern FINGERPRINT_FILE_PATTERN = Pattern.compile("[0-9a-f]{28}\\.xml");
```

Comment:
Probably better as private or package-private if the tests need it but since the class is now restricted it's fine with me as-is.